lib/msal-browser/docs/configuration.md

```diff
@@ -95,11 +93,8 @@ const msalInstance = new PublicClientApplication(msalConfig);
-| `storeAuthStateInCookie`    | If true, stores cache items in cookies as well as browser cache. Should be set to true for use cases using IE.                                                                                                                                                                                                                                                                                                                  | boolean                                                                                                 | `false`                                             |
-| `secureCookies`             | If true and `storeAuthStateInCookies` is also enabled, MSAL adds the `Secure` flag to the browser cookie so it can only be sent over HTTPS.                                                                                                                                                                                                                                                                                     | boolean                                                                                                 | `false`                                             |
-| `cacheMigrationEnabled`     | If true, cache entries from older versions of MSAL will be updated to conform to the latest cache schema on startup. If your application has not been recently updated to a new version of MSAL.js you can safely turn this off. In the event old cache entries are not migrated it may result in a cache miss when attempting to retrieve accounts or tokens and affected users may need to re-authenticate to get up to date. | boolean                                                                                                 | `true` when using `localStorage`, `false` otherwise |
-| `claimsBasedCachingEnabled` | If `true`, access tokens will be cached under a key containing the hash of the requested claims string, resulting in a cache miss and new network token request when the same token request is made with different or missing claims. If set to `false`, tokens will be cached without claims, but all requests containing claims will go to the network and overwrite any previously cached token with the same scopes.        | boolean                                                                                                 | `false`                                             |
```

Comment:
Let us ensure we test this, that we *never* cache claims. I want to make sure we have `Loop` who had this bug before, and `Teams` who I think are setting this to `false` are not broken with v5.

Reply:
@sameerag See tests added below to ensure no claims cached.

Reply:
Let's get claims removed from the cache key as well if we're removing the functionality. Doesn't need to be in this PR, or even a requisite before GA, but let's note it somewhere so we can clean it up at some point

Reply:
I'd recommend this before we GA v5.  Otherwise, it sounds like we could have some cache behavior changes after removing claims from the cache key.

Reply:
Defining a model for changes like that is what I'm working on now, we need to be able to make cache changes anytime so I don't feel strongly that it needs to happen right away but if we want to plan that in as part of the GA that's totally cool :)

Reply:
+1 to make this a gating feature for GA. @tnorling so this means you will also consider folks transitioning from claims cached to none in your work too?